### ISSUE TITLE
python<3: unable to execute 'x86_64-conda_cos6-linux-gnu-gcc': No such file or directory

### ISSUE THREAD

**Issue by nathanlcarlson** [conda-forge/python-feedstock#248]
Issue:
The following actions were taken:
```
$ wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
$ bash Miniconda3-latest-Linux-x86_64.sh -b -p /usr/local/conda
$ source /usr/local/conda/bin/activate
$ conda update conda
$ conda create -n test_cf -y 'python<3' -c conda-forge
$ conda activate test_cf
$ pip --no-cache-dir install psutil
```
Generating the following error:
```
 x86_64-conda_cos6-linux-gnu-gcc -pthread -fno-strict-aliasing -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O3 -pipe -fdebug-prefix-map=${SRC_DIR}=/usr/local/src/conda/${PKG_NAME}-${PKG_VERSION} -fdebug-prefix-map=${PREFIX}=/usr/local/src/conda-prefix -DNDEBUG -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -DPSUTIL_POSIX=1 -DPSUTIL_VERSION=551 -DPSUTIL_LINUX=1 -I/usr/local/conda/envs/test_cf/include/python2.7 -c psutil/_psutil_common.c -o build/temp.linux-x86_64-2.7/psutil/_psutil_common.o
unable to execute 'x86_64-conda_cos6-linux-gnu-gcc': No such file or directory
    error: command 'x86_64-conda_cos6-linux-gnu-gcc' failed with exit status 1
    
    ----------------------------------------
Command "/usr/local/conda/envs/test_cf/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-kLkZ0R/psutil/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-zASEj0/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-install-kLkZ0R/psutil/
```

**Note: If the `-c conda-forge` is removed in the environment creation the error does not occur.**
<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
# packages in environment at /usr/local/conda/envs/test_cf:
#
# Name                    Version                   Build  Channel
ca-certificates           2018.11.29           ha4d7672_0    conda-forge
certifi                   2018.11.29            py27_1000    conda-forge
libffi                    3.2.1             hf484d3e_1005    conda-forge
libgcc-ng                 7.3.0                hdf63c60_0    conda-forge
libstdcxx-ng              7.3.0                hdf63c60_0    conda-forge
ncurses                   6.1               hf484d3e_1002    conda-forge
openssl                   1.1.1a            h14c3975_1000    conda-forge
pip                       19.0.3                   py27_0    conda-forge
python                    2.7.15            h721da81_1007    conda-forge
readline                  7.0               hf8c457e_1001    conda-forge
setuptools                40.8.0                   py27_0    conda-forge
sqlite                    3.26.0            h67949de_1000    conda-forge
tk                        8.6.9             h84994c4_1000    conda-forge
wheel                     0.33.1                   py27_0    conda-forge
zlib                      1.2.11            h14c3975_1004    conda-forge
```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info
     active environment : test_cf
    active env location : /usr/local/conda/envs/test_cf
            shell level : 2
       user config file : /root/.condarc
 populated config files : 
          conda version : 4.6.7
    conda-build version : not installed
         python version : 3.7.1.final.0
       base environment : /usr/local/conda  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/linux-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /usr/local/conda/pkgs
                          /root/.conda/pkgs
       envs directories : /usr/local/conda/envs
                          /root/.conda/envs
               platform : linux-64
             user-agent : conda/4.6.7 requests/2.21.0 CPython/3.7.1 Linux/3.10.0-957.el7.x86_64 centos/7.6.1810 glibc/2.17
                UID:GID : 0:0
             netrc file : None
           offline mode : False

$ uname -a
Linux [hostname] 3.10.0-957.el7.x86_64 #1 SMP Thu Nov 8 23:39:32 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
# CentOS 7
```
</details>

**Comment by jjhelmus** [comment]
This was a mistake in how the `_sysconfigdata.py` file with default configuration information was copied into the package.  #249 fixes this issue.  

@pitrou I believe this is the same issue that was causing build error in Arrow.

**Comment by nathanlcarlson** [comment]
@jjhelmus Thank you for the timely response. I am glad I found the correct place to report this.

**Comment by pitrou** [comment]
Also @xhochy.

**Comment by pitrou** [comment]
#249 seems to fix the issue locally. I'll try to get CI to make a new run.

**Comment by pitrou** [comment]
Travis-CI is fixed now too. Thank you very much @jjhelmus !

**Comment by jakirkham** [comment]
Did old builds already get marked as `broken`? Should we go ahead and close this issue?

**Comment by jjhelmus** [comment]
Yes this should be closed, I'm not sure why the comment in #249 did do so automatically.

**Comment by jjhelmus** [comment]
I labeled the 2.7.15 build 1007 and 3.6.7 build 1003 package to the `broken` label on Anaconda Cloud.
